@navikt/core/react/src/timeline/hooks/useTimelineRows.ts

```diff
@@ -134,17 +134,33 @@ export const useTimelineRows = (
     [rows, startDate, endDate, direction],
   );
 
-const earliestDate = (earliest: Date, period: Period) =>
-  period.start < earliest ? period.start : earliest;
+export const useEarliestDate = ({
+  startDate,
+  rows,
+}: {
+  startDate?: Date;
+  rows: Omit<Period, "id" | "endInclusive">[][];
```

Comment:
If we require only the props we need, we don't need `@ts-expect-error` in the tests.
```suggestion
  rows: Pick<Period, "start">[][];
```